dist/tools/randsort/randsort.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2020 Sören Tempel <tempel@uni-bremen.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+import os
+import sys
+import random
+import secrets
+
+def secrets_random():
```

Comment:
It might make sense to make the "random" value user configurable (e.g., as simple command line argument that gets hashed or similar).
That way we can use a semi-fixed value for CI (as done for ssp, maybe re-use).
CI would go through all the "randomizing" but with a defined result.
The semi-fixed value could be changed regularly (weekly or before nightly builds?) so we spot possible issues caused by different linking orders.

Reply:
Sounds good, I will look into implementing a `-seed` command line argument.